docs/docusaurus/docs/conceptual_guides/gx_overview.md

```diff
@@ -0,0 +1,148 @@
+
+A Data Context defines a GX project.  In Python, the Data Context object serves as your entry point for the GX API and manages various classes to limit the objects you need to directly manage yourself.  A Data Context contains all of the metadata used by GX, the configurations for GX objects, and the output from validating data.
+
+The following are the available Data Context types:
+- **Ephemeral Data Context:** Exists in memory, and does not persist beyond the current Python session.
+- **File Data Context:** Exists as a folder and configuration files. Its contents persist between Python sessions.
+- **Cloud Data Context:** Supports persistence between Python sessions, but additionally serves as the entry point for Great Expectations Cloud.
+
+- [Data Context tutorials](/docs/guides/setup/configure_data_contexts_lp)
+
+### The GX API
+
+A Data Context object in Python provides convenience methods to further configure and interact with GX.  These methods and the objects and additional methods accessed through them compose the GX public API.  
+
+Through the GX public API you will be able to connect GX to your data sources, define the standards to which you expect your data to conform, retrieve data from your data sources, and validate that data against your defined standards.  When working with GX in Python, you will almost always start by instantiating an existing or new Data Context with which to access the GX public API.
+
+- [The GX API reference](/docs/reference/api_reference)
+
+### Settings and Stores
+
+Stores contain the metadata GX uses.  This includes things such as configurations for GX objects, information that is recorded when GX validates data, and credentials used for accessing data sources or remote environments.  GX utilizes one Store for each type of metadata, and the Data Context contains the settings that tell GX where that Store should reside and how to access it.
+
+A Data Context includes default configurations for your Store locations. The default location of Stores in a Filesystem or Cloud Data Context is nested within the same folder hierarchy that contains the Data Context itself. An Ephemeral Data Context will default to keeping your Stores in memory. You can update these defaults to specify other storage locations, such as hosting them in a shared cloud environment or external folders.
+
+- [Stores tutorials](/docs/guides/setup/setup_overview_lp)
+
+### Data Docs
+
+Data Docs are human-readable documentation generated by GX.  In general, Data Docs describe the standards that you expect your data to conform to, and the results of validating your data against those standards.  The storage and retrieval of this information is managed by the Data Context.
+
+Data Docs exist as webpages.  By default, a new Data Context includes a configuration for building them locally for File Data Contexts, in memory for Ephemeral Data Contexts, and online for Cloud Data Contexts.
```

Comment:
- Remove "Data Docs exist as webpages."
- Can we rephrase the second sentence with Data Docs as the subject. I was confused by we're back to talking about Data Context again. Or maybe for configuration of Data Docs see this link should suffice.

Reply:
I removed the second sentence entirely.